packages/appstore/src/components/modals/mt5-migration-modal/mt5-migration-modal.tsx

```diff
@@ -0,0 +1,279 @@
+        );
+    };
+
+    React.useEffect(() => {
+        if (svg_accounts_to_migrate.length) {
+            toggleMT5MigrationModal();
+        }
+    }, []);
+
+    const onConfirmMigration = () => {
+        toggleMT5MigrationModal();
+        openPasswordModal({ category: 'real', type: 'financial' });
+    };
+
+    return (
+        <div>
+            <React.Suspense fallback={<UILoader />}>
+                <DesktopWrapper>
+                    <Modal
+                        className='mt5-migration-modal'
+                        disableApp={disableApp}
+                        enableApp={enableApp}
+                        exit_classname='cfd-modal--custom-exit'
+                        is_open={is_mt5_migration_modal_open}
+                        title={modal_title}
+                        toggleModal={toggleMT5MigrationModal}
+                        width='58.8rem'
+                        height={
+                            show_modal_front_side
+                                ? no_of_svg_accounts_to_migrate < 1
+                                    ? '54.2rem'
+                                    : '44rem'
+                                : '61.6rem'
+                        }
+                    >
+                        {show_modal_front_side ? <ShowFrontSide /> : <ShowBackSide />}
+
+                        {show_modal_front_side ? (
+                            <Modal.Footer has_separator>
+                                <Button type='button' large primary onClick={() => setShowModalFrontSide(false)}>
+                                    Next
+                                </Button>
+                            </Modal.Footer>
+                        ) : (
+                            <Modal.Footer has_separator>
+                                <Button type='button' large secondary onClick={() => setShowModalFrontSide(true)}>
+                                    Back
+                                </Button>
+
+                                <Button type='button' large primary onClick={onConfirmMigration}>
+                                    Next
+                                </Button>
+                            </Modal.Footer>
+                        )}
+                    </Modal>
+                </DesktopWrapper>
+                <MobileWrapper>
+                    <MobileDialog
+                        wrapper_classname='mt5-migration-modal'
+                        disableApp={disableApp}
+                        enableApp={enableApp}
+                        portal_element_id='deriv_app'
+                        title={modal_title}
+                        visible={is_mt5_migration_modal_open}
+                        toggleModal={toggleMT5MigrationModal}
+                    >
+                        <Div100vhContainer height_offset={'150px'}>
+                            {show_modal_front_side ? <ShowFrontSide /> : <ShowBackSide />}
+                        </Div100vhContainer>
```

Comment:
Why are you wrapping only this part in `Div100vhContainer`?

Reply:
I only need the Modal Content to be in 100vh. Footer I will have to remain as fixed bottom